src/routes/DisciplineSelectionScreen.tsx

```diff
@@ -23,9 +23,9 @@ const ItemText = styled.View`
   align-items: center;
 `
 
-const StyledList = styled(FlatList as new () => FlatList<DisciplineType>)`
+const StyledList = styled(FlatList)`
   width: 100%;
-`
+` as ComponentType as new () => FlatList<DisciplineType>
```

Comment:
previous solution didn't work anymore

Reply:
why do we need the `as Flatlist` anyway?

Reply:
To type the generic